axelor-crm/src/main/resources/i18n/messages_fr.csv

```diff
@@ -491,6 +495,7 @@
 "Select Partner","Sélectionner un tiers",,
 "Select existing contact","Sélectionner un contact existant",,
 "Select existing partner","Sélectionner un tiers existant",,
+"Select tier type to create:","Selectionner le type de tier à créer:",,

```

Comment:
please change the key to "Select partner type to create:" and change the translation to "Sélectionner le type de tiers à créer :"